src/typechecker/FStar.TypeChecker.Normalize.fst

```diff
@@ -625,6 +625,22 @@ let filter_out_lcomp_cflags flags =
 
 let closure_as_term cfg env t = non_tail_inline_closure_env cfg env t
 
+let binder_closure_as_term cfg env (b : binder) =
+  let x = { b.binder_bv with sort = closure_as_term cfg env b.binder_bv.sort } in
+  let imp = match b.binder_qual with
+            | Some (S.Meta t) -> Some (S.Meta (closure_as_term cfg env t))
+            | i -> i in
+  let attrs = List.map (closure_as_term cfg env) b.binder_attrs in
+  S.mk_binder_with_attrs x imp b.binder_positivity attrs
+
+let binders_closure_as_term cfg env (bs : binders) =
```

Comment:
Is this different from `close_binders` above?

Reply:
I completely missed that since it didn't have `closure_as` in the name!  I have removed the duplicate.